src/main/java/org/mustangproject/ZUGFeRD/ZUGFeRDExporter.java

```diff
@@ -478,17 +479,27 @@ public void PDFAttachGenericFile(PDDocument doc, String filename, String relatio
 		doc.getDocumentCatalog().setNames(names);
 
 		// AF entry (Array) in catalog with the FileSpec
-		COSArray cosArray = (COSArray) doc.getDocumentCatalog().getCOSObject().getItem("AF");
-		if (cosArray == null) {
-			cosArray = new COSArray();
-		}
-		cosArray.add(fs);
-		COSDictionary dict2 = doc.getDocumentCatalog().getCOSObject();
-		COSArray array = new COSArray();
-		array.add(fs.getCOSObject()); // see below
-		dict2.setItem("AF", array);
```

Comment:
This line in the current mustangproject code effectively has no effect. the "AF" object is overwritten on the next line.